.github/workflows/cr.yml

```diff
@@ -0,0 +1,35 @@
+name: Continuous Releases
+
+on:
+  push:
+    branches:
+      - main
+      - docusaurus-v**
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  release:
+    name: Continuous Releases
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+      - name: Installation
+        run: yarn
+
+      - name: Build packages
+        run: yarn build:packages
+
+      - name: Fresh templates
+        run: |
+          yarn create-docusaurus fresh-examples/docusaurus-classic classic --javascript -p npm -s 
+          yarn create-docusaurus fresh-examples/docusaurus-classic-typescript classic --typescript -p npm -s
+
+      - name: Release
+        run: npx pkg-pr-new publish './packages/*' --template './fresh-examples/*' --compact --comment=off
```

Comment:
I also turned off comments since there are many packages, but still, on each commit, we'll have the Continuous Releases check run.

![image](https://github.com/user-attachments/assets/7f17262f-9751-4408-b368-756d6f7b2529)

When clicking details, we can see all the packages there without the need for a comment on each pull request.

Reply:
Thanks

I think the comment is helpful for discovery, but it has to be more concise to be effective and only post the 2 template links. Until you have a better comment, it's preferable to hide it.

Reply:
Thanks

I think the comment is helpful for discovery, but it has to be more concise to be effective and only post the 2 template links. Until you have a better comment, it's preferable to hide it.